silk-react-components/src/HierarchicalMapping/Components/MessageHandler.jsx

```diff
@@ -3,111 +3,99 @@ import React from 'react';
 import _ from 'lodash';
 import rxmq from 'ecc-messagebus';
 
```

Comment:
can you also think about some file restructuring?
E.g. this file does not really belong to HM as it is a generic notification displayer.
Other examples would be `helpers.js vs. utils/` or` SharedComponents.jsx` which is a mix of a couple of non-related react components and `Suggestion.js`